packages/manager/apps/web/client/app/hosting/database/order/public/components/steps/db-categories-offers/controller.js

```diff
@@ -17,7 +17,18 @@ export default class WebHostingDatabaseOrderComponentsDbCategoriesOffersControll
   }
 
   getDbEngines() {
-    return this.model.dbCategory?.selectVersion?.engines || [];
+    const engines = this.model.dbCategory?.selectVersion?.engines;
+    if (engines && engines.length) {
```

Comment:
```suggestion
    if (engines?.length) {
```
Sonar tags that like code smells